app/routes/groups_.$groupId.tsx

```diff
@@ -117,21 +120,27 @@ export default function GroupRoute() {
         <div className="min-w-[280px] min-h-[280px] p-2 md:p-10 md:flex md:flex-row">
           <div className="md:w-1/2 p-2 md:p-4">
             <h2 className="font-extrabold md:text-2xl">What we are about</h2>
-            <p>{data.group.description}</p>
+            <p>{group.description}</p>
           </div>
           <div className="md:w-1/2 p-2 md:p-4 md:border-secondary md:border-2 rounded-lg">
             <h2 className="font-extrabold md:text-2xl">Upcoming Events</h2>
             <div className="flex flex-col">
-              {!data.group.events ? (
+              {!group.events || group.events.length === 0 ? (
```

Comment:
I think you made it so `group.events` is always an array now, so we could remove the `!group.events` check?

Reply:
I think it's good to be guarded, even when it's always array

Reply:
NITs:
- I agree that `group.events` will always be `true`.
- To be more concise, it is possible to use just check `group.events.length` instead of `group.events.length === 0`, since `group.events.length` will be `true` if it's greater than 0.